docs/.gitignore

```diff
@@ -1,38 +0,0 @@
-# Dependencies
-/node_modules
-
-# Production
-build
-!docs/build
-
-# Generated files
-.docusaurus
-.cache-loader
-versioned_docs
-versioned_sidebars
-docs/node_modules
-docs/docs/modules
-docs/docs/spec
-docs/docs/architecture
-docs/docs/rfc
-docs/docs/tooling/01-cosmovisor.md
-docs/docs/tooling/02-confix.md
-docs/docs/tooling/03-hubl.md
-docs/docs/core/17-autocli.md
-docs/docs/packages/01-depinject.md
-docs/docs/packages/02-collections.md
-docs/docs/packages/03-orm.md
-docs/user/run-node/04-rosetta.md
-docs/build/migrations/02-upgrading.md
-docs/develop/advanced/17-autocli.md
```

Comment:
is it okay to remove this?

Reply:
If we keep the script, then no, we should keep this.